docs/userguide/usage_tracking.rst

```diff
@@ -21,13 +21,13 @@ Choose the data you share with Usage Tracking Levels.
 
 **Usage Tracking Levels:**
 
-* **Level 1:** Basic information such as Python version, Parsl version, and platform details.
-* **Level 2:** Configuration information including provider, executor, and launcher names.
-* **Level 3:** Workflow execution details, including the number of applications run, failures, and execution time.
+* **Level 1:** Only basic information such as Python version, Parsl version, and platform details (Linux, MacOS etc.).
```

Comment:
Is platform details just OS? If so, say so...  This is still too vague to me

Reply:
Yes. It's just the OS name. I'll make the change